locales/ca/generate.json

```diff
@@ -0,0 +1,52 @@
+{
+   "meta": {
+      "description": "Genera llistes amb les cançons que les teves bandes estan girant actualment",
+      "keywords": "Spotify, Setlist.fm, Generador de llistes, Festivals, Música, Concerts",
+      "title": "Generar | Festwrap"
+   },
+   "steps": {
+      "navigation": {
+         "finish": "Finalitzar",
+         "generate": "Generar",
+         "next": "Següent",
+         "previous": "Anterior"
+      },
+      "step1": {
+         "description": "Crea una llista nova o utilitza una llista propia existent",
+         "form": {
+            "createNewPlaylist": {
+               "description": "Utilitza aquesta opció per crear una nova llista de reproducció a Spotify",
+               "giveAName": "Dona un nom a la llista",
+               "namePlaceholder": "Nom de la llista",
+               "privatePlaylist": {
+                  "description": "La llista de reproducció serà privada i només tu podràs veure-la",
+                  "title": "Llista privada"
+               },
+               "title": "Crear una nova llista"
+            },
+            "useExistingPlaylist": {
+               "description": "Utilitza aquesta opció per afegir cançons a una de les teves llistes de reproducció ja existent",
+               "selectPlaylist": "Selecciona una llista de reproducció",
+               "title": "Utilitzar una llista existent"
+            }
+         },
+         "title": "Tria una opció per començar"
```

Comment:
I would change this to be "Configura la llista" or similar. It is more specific.